core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/run_sh.go

```diff
@@ -253,7 +252,6 @@ func getCommandToRun(builtin *RunShCapabilities) (string, error) {
 	if err != nil {
 		return "", stacktrace.Propagate(err, "An error occurred while replacing runtime values in run_sh")
 	}
-	commandWithNoNewLines := strings.ReplaceAll(maybeSubCommandWithRuntimeValues, newlineChar, " ")
```

Comment:
i don't know why we did this though?

Reply:
ah we have a failing test for this

Reply:
My theory is that while writing the tests someone saw that `echo kurtosis` would echo

```
kurtosis

```

and that messed up the first test so they changed it to remove new lines

Reply:
@leoporoli fysa

Reply:
@h4ck3rk3y you conclusion makes sense